### PR TITLE
Cleaning and reorganizing concept model page for CVD

### PR DESCRIPTION
**Pull request by SylLutze**
Adding links for different pages, significant reorganization, streamlining of text and information, adding outstanding todos throughout

### CODE REVIEW

**Review by aflaxman — APPROVED** [review]

File: docs/source/concept_models/vivarium_us_cvd/concept_model.rst [hunk 1]
```diff
@@ -74,235 +74,281 @@ Vivarium - US Health Disparities - CVD Interventions
...
-.. [Munoz-NEJM]_ 
+.. _4.1.3:
 
-.. **Scenarios:**
-.. 	- Polypill 1.0: All individuals with IHD, past ischemic stroke, or prevalent PAD (who are not already on medications) receive atorvastatin (10 mg), amlodipine (2.5 mg), losartan (25 mg), and hydrochlorothiazide (12.5 mg) regardless of SBP or LDL-c levels. Medications lead to expected reduction in SBP and LDL-c. Adherence is increased 13% following receipt of those medications.  
-.. 	- Polypill 0.5: Above scenario with, but a reduction to 50% of individuals meeting criteria receiving prescription for atorvastatin (10 mg), amlodipine (2.5 mg), losartan (25 mg), and hydrochlorothiazide (12.5 mg) regardless of SBP or LDL-c levels). Medications lead to expected reduction in SBP and LDL-c. Adherence among individuals receiving therapy is increased 13% following receipt of those medications. 
+4.1.2 Risk Effects Models
```
Comment (stevebachmeier) [inline docs/source/concept_models/vivarium_us_cvd/concept_model.rst:246]: should be 4.1.3?

File: docs/source/concept_models/vivarium_us_cvd/concept_model.rst [hunk 2]
```diff
@@ -74,235 +74,281 @@ Vivarium - US Health Disparities - CVD Interventions
...
+#. **Medical Outreach 50% Coverage**  
+#. **Medical Outreach 100% Coverage**
+#. **Polypill 50% Coverage**  
+#. **Polypill 100% Coverage**
+#. **Lifestyle Modification 50% Coverage**
+#. **Lifestyle Modification 100% Coverage**
+#. **Combination of All Modifications 50% Coverage**
+#. **Combination of All Modifications 100% Coverage**
 
+**Baseline** is assumed to have no one is enrolled in any intervention.
```
Comment (plinds) [inline docs/source/concept_models/vivarium_us_cvd/concept_model.rst:147]: Typo: to have no one enrolled (remove the is before enrolled)

File: docs/source/concept_models/vivarium_us_cvd/concept_model.rst [hunk 3]
```diff
@@ -74,235 +74,281 @@ Vivarium - US Health Disparities - CVD Interventions
...
+#. **Lifestyle Modification 50% Coverage**
+#. **Lifestyle Modification 100% Coverage**
+#. **Combination of All Modifications 50% Coverage**
+#. **Combination of All Modifications 100% Coverage**
 
+**Baseline** is assumed to have no one is enrolled in any intervention. 
 
-.. **Mediators (M)**:
+**Medical Outreach 50% Coverage** assumes 50% of eligible simulants are enrolled in the outreach intervention. Scales 
+linearly over 1 year such that there is 0% coverage at baseline and 50% at year 1.
```
Comment (plinds) [inline docs/source/concept_models/vivarium_us_cvd/concept_model.rst:150]: Let's add that we want to hold at 50% coverage for the remainder of the sim (add this extension to all scenario descriptions)

File: docs/source/concept_models/vivarium_us_cvd/concept_model.rst [hunk 4]
```diff
@@ -74,235 +74,281 @@ Vivarium - US Health Disparities - CVD Interventions
...
+**Lifestyle Modification 100% Coverage** assumes all eligible simulants are enrolled in the lifestyle modeification 
+intervention. Scales linearly over 1 year such that there is 0% coverage at baseline and 100% at year 1. 
 
-4.0 Intervention
-++++++++++++++++
+**Combination of All Modifications 50% Coverage** assumes 50% coverage for all interventions, implemented 
+simultaneously. Scales linearly over 1 year such that there is 0% coverage at baseline and 50% at year 1.
 
-.. _uscvd4.1:
+**Combination of All Modifications 50% Coverage** assumes 100% coverage for all interventions, implemented
```
Comment (plinds) [inline docs/source/concept_models/vivarium_us_cvd/concept_model.rst:170]: Title typo: *100% coverage

File: docs/source/concept_models/vivarium_us_cvd/concept_model.rst [hunk 5]
```diff
@@ -74,235 +74,281 @@ Vivarium - US Health Disparities - CVD Interventions
...
+.. list-table:: Developmental model CVD simulation timeframe and intervention dates
+  :header-rows: 1
 
-**Scenarios:**
+  * - Parameter
+    - Value
+  * - Date of simulation burn-in period start
+    - N/A (no burn-in)
+  * - Date of simulation observation period start
+    - January 1, 2019
```
Comment (plinds) [inline docs/source/concept_models/vivarium_us_cvd/concept_model.rst:193]: I'm assuming we might  push this to Jan 1 2023 at this point, but TBD

**Review by plinds — APPROVED** [review]

**Pull request merged**